command/git_push.go

```diff
@@ -0,0 +1,153 @@
+package command
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/evergreen-ci/evergreen/model"
+	"github.com/evergreen-ci/evergreen/rest/client"
+	restModel "github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/mitchellh/mapstructure"
+	"github.com/mongodb/grip/level"
+	"github.com/pkg/errors"
+)
+
+// gitMerge is a command that commits tracked changes and pushes the resulting commit to a remote repository
+type gitPush struct {
```

Comment:
I think this could use a DryRun mode. People adding this to their yml will need to test it, but don't want to actually push anything

Reply:
This also depends if we intend this to be applicable outside the commit queue.

Reply:
done. Added a flag that does everything except push back up to GitHub